objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/NeuralNetTestBase.java

```diff
@@ -19,6 +19,7 @@
 
 public class NeuralNetTestBase extends ObjRecTestBase {
     public static final String MODEL_DIRECTORY = "src/test/resources/models";
+    public static final String SOURCE_NAME = "src";
```

Comment:
tweaky, but having this name match where we keep our source bothers me :-).  Maybe a different name.  Im concerned that if someone confuses things or their setting, `objectRecognition/src/...` looks an awful lot like where our repo keeps code...

maybe use testSourceName as the name or something.

Reply:
Sounds good, changing this now!